### PR TITLE
Add external minio support

### PR DESCRIPTION
**Pull request by sthapa**
I checked the servicex app code and it looks like this should work but I'm doing a quick test that should confirm this later this afternoon.

### CODE REVIEW

**Comment by BenGalewsky** [comment]
Not quite so easy - you need to prevent the helm chart from deploying a minio instance

**Comment by sthapa** [comment]
@BenGalewsky this now disables the minio pod deploy and should set the various settings.  Mind taking a quick glance at it while I test?

File: servicex/templates/app/configmap.yaml [hunk 1]
```diff
@@ -103,23 +103,32 @@ data:
 
     {{ if .Values.objectStore.enabled }}
     OBJECT_STORE_ENABLED = True
+      {{ if not .Values.objectStore.internal }}
+    # using external minio 
+    MINIO_URL = '{{ .Values.objectStore.publicURL }}'
+    MINIO_URL_TRANSFORMER = '{{ .Values.objectStore.publicURL }}'
+    MINIO_PUBLIC_URL = '{{ .Values.objectStore.publicURL }}'
+    MINIO_SECURED = {{ ternary "True" "False" .Values.objectStore.usetls }}
```
Comment (BenGalewsky) [inline servicex/templates/app/configmap.yaml:111]: This is eventually a python config file. You should remove the quotes around `True` and `False`

**Review by BenGalewsky — CHANGES_REQUESTED** [review]
Please update `docs/deployment/reference.md` to show the new settings

File: servicex/templates/app/configmap.yaml (hunk 1 above)
Comment (sthapa) [reply servicex/templates/app/configmap.yaml:111]: Without the quotes, helm looks for a function called True and errors out.  The configmap gets the strings without the quotes.

Comment (sthapa) [reply servicex/templates/app/configmap.yaml:111]: Also updated the docs with the new settings

Comment (BenGalewsky) [reply servicex/templates/app/configmap.yaml:111]: I don't see any changes to the [deployment reference docs](https://github.com/ssl-hep/ServiceX/blob/b28498a96a5df99f57ac0a261ea679175f02cf70/docs/deployment/reference.md?plain=1#L8)

File: servicex/values.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ secrets:
 # want to have the option of delivering results as parquet files
 objectStore:
   enabled: true
+  internal: false
   publicURL:
```
Comment (BenGalewsky) [inline servicex/values.yaml:15]: This should default to `true` to keep existing behavior. It will break all current deployments otherwise

Comment (BenGalewsky) [reply servicex/values.yaml:15]: Pushed a fix

File: servicex/templates/app/configmap.yaml (hunk 1 above)
Comment (BenGalewsky) [reply servicex/templates/app/configmap.yaml:111]: Pushed an update

**Review by BenGalewsky — APPROVED** [review]

**Pull request merged**